### PR TITLE
Feat/submit_clinical_ FileNavigator section skeleton 

### PR DESCRIPTION
**Pull request by ciaranschutte**
-  FileNavigator section skeleton of submit clinical page

Going into feature branch

Context for review:
commented out sections remain because they are currently being worked on in scope of this logic 
vs slowly copying over all code and having to comment out even more

changes are to fix types or invalid code - anything else stylistic is copied over and not new code

### CODE REVIEW

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FileRecordTable.tsx [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+//         }
+//         columns={tableColumns}
+//         data={tableData}
+//         showPagination
+//       />
+//     </div>
+//   );
+// };
+
+// export default FileRecordTable;
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FileRecordTable.tsx:392]: Is this file necessary?

Comment (ciaranschutte) [reply src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FileRecordTable.tsx:392]: yes

Comment (demariadaniel) [reply src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FileRecordTable.tsx:392]: I see, in description

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FilePreview.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { useSubmissionSystemStatus } from "@/app/hooks/useSubmissionSystemStatus";
+import { css } from "@/lib/emotion";
+import { Button, Typography } from "@icgc-argo/uikit";
+import { FC, SyntheticEvent } from "react";
+import { ClinicalSubmissionEntity, ClinicalSubmissionState } from "../../types";
+
+type FilePreivewProps = {
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/FilePreview.tsx:26]: Should be `FilePreviewProps`

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import { Icon, VerticalTabs } from "@icgc-argo/uikit";
+import { FC, SyntheticEvent } from "react";
+import { ClinicalEntities, ClinicalSubmissionEntity } from "../../types";
+
+export const VerticalTabsSection: FC<{
+  fileStates: ClinicalEntities;
+  selectedFile: ClinicalSubmissionEntity | undefined;
+  onFileSelect: (clinicalType: string) => void;
+}> = ({ fileStates, selectedFile, onFileSelect }) => {
+  console.log("v tabs", fileStates);
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/Tabs.tsx:30]: Should keep our console clean

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/index.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+}: {
+  submissionState: ClinicalSubmissionState;
+  fileStates: ClinicalEntities;
+  clearDataError: (file: ClinicalSubmissionEntity) => Promise<any>;
+  selectedClinicalEntityType: string;
+  onFileSelect: (clinicalEntityType: string) => void;
+  submissionVersion: ClinicalSubmission["version"];
+  programShortName: ClinicalSubmission["programShortName"];
+}) => {
+  console.log("file navigaotr file staes", fileStates);
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/index.tsx:53]: Should be removed as well

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import { ClinicalSubmissionQuery } from "@/__generated__/graphql";
+
+// sometimes we use GQL data types other things we use the converted local types, they will conflict
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts:21]: This is a bit hard to understand, I think `other things` is a typo?

**Comment by ciaranschutte** [comment]
all feedback addressed https://github.com/icgc-argo/rdpc-ui/pull/87/commits/7802221b0bbae6e7e937450332f8831bf152f7e6

**Pull request merged**